### PR TITLE
[11.0][MIG] partner_contact_job_position

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

Changed dependency.

### CODE REVIEW

File: partner_contact_job_position/__manifest__.py [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
 # Copyright 2015 Jairo Llopis <jairo.llopis@tecnativa.com>
 # Copyright 2017 David Vidal <david.vidal@tecnativa.com>
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 {
-    "name": "Partner job position",
+    "name": "Partner Job Position",
     "summary": "Categorize job positions for contacts",
-    "version": "10.0.1.0.1",
+    "version": "11.0.1.0.1",
```
Comment (pedrobaeza) [inline partner_contact_job_position/__manifest__.py]: 11.0.1.0.0

File: partner_contact_job_position/__manifest__.py [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
     'category': 'Customer Relationship Management',
-    "website": "http://www.antiun.com",
+    "website": "https://github.com/OCA/partner-contact",
     'author': 'Serv. Tecnolog. Avanzados - Pedro M. Baeza, '
```
Comment (pedrobaeza) [inline partner_contact_job_position/__manifest__.py]: Remove old companies (Antiun & Servicios...)

File: partner_contact_job_position/README.rst [hunk 3]
```diff
@@ -28,23 +28,24 @@ To use this module, you need to:
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/134/10.0
+   :target: https://runbot.odoo-community.org/runbot/134/11.0
```
Comment (pedrobaeza) [inline partner_contact_job_position/README.rst]: No new README by fragments?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]
When CIs goes to green

File: partner_contact_job_position/README.rst (hunk 3 above)
Comment (MiquelRForgeFlow) [reply partner_contact_job_position/README.rst]: I was waiting your super helpful comment of the other PR

Comment (pedrobaeza) [reply partner_contact_job_position/README.rst]: :+1:

File: partner_contact_job_position/README.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+having them categorized in tree form and allow you to choose a categorized job
+position for your contacts in addition to the current custom string for job
+position.
+
+Configuration
+=============
+
+To configure job positions, you need to:
+
+* Go to *Contacts > Configuration > Address Book > Job Positions*.
```
Comment (LoisRForgeFlow) [inline partner_contact_job_position/README.rst]: remove address book from the path

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM, minor comment

**Comment by lfreeke** [comment]
:+1: (test)

**Pull request merged**